javascript/builtins/AggregateError.json

```diff
@@ -80,6 +80,47 @@
               "deprecated": false
             }
           }
+        },
+        "serializable_object": {
+          "__compat": {
+            "description": "<code>AggregateError</code> is serializable",
+            "mdn_url": "https://developer.mozilla.org/docs/Glossary/Serializable_object",
```

Comment:
PS. I added mdn link to serializable object info for these cases so now it is clear what it means.